### PR TITLE
Enable V3 publishing

### PR DESCRIPTION
**Pull request by NikolaMilosavljevic**
Fixes issues with publishing in this repo.

Verified with private internal build.

### CODE REVIEW

**Review by riarenas — APPROVED** [review]
LGTM

**Comment by riarenas** [comment]
Looks like my review doesn't count, but this looks correct.

**Comment by NikolaMilosavljevic** [comment]
Close and reopen to trigger checks after updating the PR.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -53,10 +53,10 @@ stages:
           # For public or PR jobs, use the hosted pool.  For internal jobs use the internal pool.
           # Will eventually change this to two BYOC pools.
           ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
-            name: Hosted VS2017
+            name: Hosted VS2019
```
Comment (riarenas) [inline azure-pipelines.yml:56]: Hmm, this syntax is pretty pretty old and I don't think it's supported anymore.

```suggestion
            vmImage: windows-2019
```

This is the reason PR validation is not running. It's trying to be queued but having pool problems: https://dnceng.visualstudio.com/public/_build/results?buildId=1962467&view=results

Comment (riarenas) [reply azure-pipelines.yml:56]: [More details on the vmImage syntax](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml#use-a-microsoft-hosted-agent)

**Review by vijayrkn — APPROVED** [review]

**Pull request merged**